elasticsearch-transport/lib/elasticsearch/transport/client.rb

```diff
@@ -125,6 +125,11 @@ def perform_request(method, path, params={}, body=nil)
         transport.perform_request method, path, params, body
       end
 
+      # Shuts down the client
+      def close
+        @transport.__close_connections
```

Comment:
if this is a public api for the transport, it seems wrong to be calling an method prefixed by __